TERMSOFSERVICE.md

```diff
@@ -105,38 +147,28 @@ obligations.
 5.5. You shall ensure that the content you provide is free of third-party
 rights and that its publication does not violate applicable law in any form.
 
-5.6. Where we suggest content, measures or services, you are obliged to examine
-their legal admissibility.
-
-5.7. Insofar as we provide you with legal models, e.g. usage and data
-protection terms, these should be considered as templates. It is your
-responsibility to adapt the models to your own legal requirements.
-
 # 6. Remuneration
 
 6.1. The remuneration currently owed is based on the chosen plan at
-https://app.tideways.io/  and is quoted net. Statutory VAT applicable at
-the time of invoicing is charged in addition.
+https://tideways.com, https://tideways.io/ or https://tideways.de and is quoted
+net. Statutory VAT applicable at the time of invoicing is charged in addition.
 
 6.2. The remuneration is due to payment at the beginning of each payment
-interval.
+interval. You may settle the remuneration using one of the payment methods we
+have offered to you. You will be provided with the invoices electronically.
 
-6.3. The terms and conditions of involved payment services (e.g. paypal) apply
-additionally.
+6.3. [Removed]
 
-6.4. We reserve the right to change the scale of remuneration at any time. With
-regard to current contracts, such change shall take effect only from the
-beginning of the following contracting period.
+6.4. [Removed]
 
-6.5. You may settle the remuneration using one of the payment methods we have
-offered to you. You will be provided with the invoices electronically.
+6.5. [Removed]
 
 6.6. If we cannot provide our services due to lacking and inadequate
 co-operation or supplies from you or if we can provide our services only by
 incurring additional expenditure, we are entitled to claim such necessary
 additional expenditure from you.
 
-6.7. Should it not be possible to have an invoice paid by direct debit, you
+6.7. Should it not be possible to have an invoice paid by credit card or direct debit, you
```

Comment:
Added missing payment method credit card that we use for most of our business with customers.